corehq/apps/app_manager/xform.py

```diff
@@ -621,6 +626,12 @@ def audio_references(self):
     def video_references(self):
         return self.media_references(form="video")
 
+    def all_references(self, lang):
+        images = self.media_references_by_lang(lang=lang, form="image")
+        video = self.media_references_by_lang(lang=lang, form="video")
+        audio = self.media_references_by_lang(lang=lang, form="audio")
```

Comment:
should this include [`video-inline`](https://github.com/dimagi/vellum/blob/master/src/javaRosa/util.js#L16)?